src/common/ssl/SSLConfig.cpp

```diff
@@ -18,6 +18,7 @@ namespace nebula {
 std::shared_ptr<wangle::SSLContextConfig> sslContextConfig() {
   auto sslCfg = std::make_shared<wangle::SSLContextConfig>();
   sslCfg->addCertificate(FLAGS_cert_path, FLAGS_key_path, FLAGS_password_path);
+  sslCfg->clientVerification = folly::SSLContext::VerifyClientCertificate::DO_NOT_REQUEST;
```

Comment:
Does this mean that the certificate from the client could be omitted?

Reply:
how about using "always" and failed if verification fails

Reply:
Not, it just check CA of server (which is new feature after library upgrading), so I disable it to keep same with before.